docs/docsite/rst/user_guide/playbooks_tags.rst

```diff
@@ -1,227 +1,402 @@
-combining it with ``--list-tasks``::
+You can apply the same tag to more than one individual task. This example tags several tasks with the same tag, "ntp":
 
-    ansible-playbook example.yml --tags "configuration,packages" --list-tasks
+.. code-block:: yaml
 
-.. warning::
-    * Fact gathering is tagged with 'always' by default. It is ONLY skipped if
-      you apply a tag and then use a different tag in ``--tags`` or the same
-      tag in ``--skip-tags``.
-
-.. _tag_reuse:
-
-Tag Reuse
-```````````````
-You can apply the same tag to more than one task. When a play is run using
-the ``--tags`` command-line option, all tasks with that tag name will be run.
-
-This example tags several tasks with one tag, "ntp"::
-
-    ---
-    # file: roles/common/tasks/main.yml
-
-    - name: be sure ntp is installed
-      yum:
-        name: ntp
-        state: present
-      tags: ntp
-
-    - name: be sure ntp is configured
-      template:
-        src: ntp.conf.j2
-        dest: /etc/ntp.conf
-      notify:
-      - restart ntpd
-      tags: ntp
-
-    - name: be sure ntpd is running and enabled
-      service:
-        name: ntpd
-        state: started
-        enabled: yes
-      tags: ntp
+   ---
+   # file: roles/common/tasks/main.yml
+
+   - name: be sure ntp is installed
+     yum:
+       name: ntp
+       state: present
+     tags: ntp
+
+   - name: be sure ntp is configured
+     template:
+       src: ntp.conf.j2
+       dest: /etc/ntp.conf
+     notify:
+     - restart ntpd
+     tags: ntp
+
+   - name: be sure ntpd is running and enabled
+     service:
+       name: ntpd
+       state: started
+       enabled: yes
+     tags: ntp
+
+   - name: be sure file sharing is installed
+     yum:
+       name:
+       - nfs-utils
+       - nfs-util-lib
+       state: present
+     tags: filesharing
+
+If you ran these four tasks in a playbook with ``--tags ntp``, Ansible would run the three tasks tagged ``ntp`` and skip the one task that does not have that tag.
+
+.. _tags_on_includes:
+
+Adding tags to includes
+-----------------------
+
+You can apply tags to dynamic includes in a playbook. As with tags on an individual task, tags on an ``include_*`` task apply only to the include itself, not to any tasks within the included file or role. If you add ``mytag`` to a dynamic include, then run that playbook with ``--tags mytag``, Ansible runs the include itself, runs any tasks within the included file or role tagged with ``mytag``, and skips any tasks within the included file or role without that tag. See :ref:`selective_reuse` for more details.
+
+You add tags to includes the same way you add tags to any other task:
+
+.. code-block:: yaml
+
+   ---
+   # file: roles/common/tasks/main.yml
+
+   - name: dynamic re-use of database tasks
+     include_tasks: db.yml
+     tags: db
+
+You can add a tag only to the dynamic include of a role. In this example, the ``foo`` tag will *NOT* apply to tasks inside the ``bar`` role::
```

Comment:
Already have bold emphasis, so might be a good idea to remove the all caps emphasis.